docs/content/concepts/layer/control/catalog/sp800-53rev5-example.md

```diff
@@ -0,0 +1,339 @@
+---
+title: SP800-53 Control Example
+heading: "NIST SP 800-53 rev 5 Annotated Example"
+weight: 30
+sidenav:
+  title: SP800-53 Control Example
+toc:
+  enabled: true
+---
+
+
+{{<callout>}}**Please note:**
+
+- The following commentary uses and shows XML syntax. A later revision will offer JSON and YAML translations.
+- Full versions of the NIST SP 800-53 Rev. 5 catalog are available in OSCAL [XML]({{< param "contentRepoPath" >}}/nist.gov/SP800-53/rev5/xml/NIST_SP-800-53_rev5_catalog.xml), [JSON]({{< param "contentRepoPath" >}}/nist.gov/SP800-53/rev5/json/NIST_SP-800-53_rev5_catalog.json), and [YAML]({{< param "contentRepoPath" >}}/nist.gov/SP800-53/rev5/yaml/NIST_SP-800-53_rev5_catalog.yaml) formats in the OSCAL content GitHub repository. These examples also include assessment and objective content from NIST SP 800-53A Rev. 5, *not shown here*.
+{{</callout>}}
+
+The following is a non-normative, partial illustration showing how control **AC-1** from NIST SP [800-53 Rev. 5](https://doi.org/10.6028/NIST.SP.800-53r5) can be rendered in OSCAL Catalog XML format with a `<control>` element.
+
+First we start with the following control text from SP 800-53. (Formatting adjusted for conciseness and legibility.)
+
+> **AC-1 POLICY AND PROCEDURES**
```

Comment:
AC-1 title is not all caps in oscal-content 800-53rev5 catalog.
```
<control class="SP800-53" id="ac-1">
         <title>Policy and Procedures</title>
```

Reply:
Indeed, very acutely observed. The PDF of the control catalog as published (the text we are transcribing) has this:

![image](https://user-images.githubusercontent.com/1588228/174178451-23c1c142-22ad-4f4c-a834-1ed0ece995fc.png)

while in the OSCAL, the values are normalized to English sentence case (which has a couple of important advantages).

Is this worth calling out in the commentary?

Reply:
> Is this worth calling out in the commentary?

Probably not. Anyone who thinks that the PDF _rendition_ is authoritative (as opposed to its lamentably unstructured content) will insist that all caps, small caps, underlines, italics, indentation, color, and other such typographical aspects convey deep meaning which cannot be ignored: Section 508 is for other folks.

Reply:
Indeed, the ornamental capitals are only one example of a rather large class of signifiers.